src/api/ledger/transaction.js

```diff
@@ -78,11 +78,22 @@ function getTransactionAsync(identifier: string, options: TransactionOptions,
     }
   }
 
+
+  /* eslint-disable no-unused-vars, handle-callback-err */
```

Comment:
> Also, why is the error case not being handled?

There is no point to check it. `getLedgerSequence` returns error only in one case - servers was not specified in `Remote` constructor, and in this case error will be thrown much earlier.